types/three/three-core.d.ts

```diff
@@ -6118,10 +6118,10 @@ export namespace SceneUtils {
 }
 
 export namespace ShapeUtils {
-    export function area(contour: number[]): number;
-    export function triangulate(contour: number[], indices: boolean): number[];
-    export function triangulateShape(contour: number[], holes: any[]): number[];
-    export function isClockWise(pts: number[]): boolean;
+    export function area(contour: Vector[]): number;
```

Comment:
Judging from the docs and the implementation `area` seems to require a Vector2, see: https://github.com/mrdoob/three.js/blob/master/src/extras/ShapeUtils.js#L16